src/Atmos/Model/AtmosModel.jl

```diff
@@ -263,4 +253,15 @@ function init_state!(m::AtmosModel, state::Vars, aux::Vars, coords, t, args...)
   m.init_state(state, aux, coords, t, args...)
 end
 
+boundary_flux_diffusive!(nf::CentralNumericalFluxDiffusive, atmos::AtmosModel,
+                         F⁺, state⁺, diff⁺, aux⁺, n⁻,
```

Comment:
Why `n⁻` instead of just `n`?

Reply:
It's the normal with respect to the minus side of the face as opposed to the normal with respect to the plus side (which is equal and opposite).

Reply:
I though it was the minus face of the side wrt to the normal?

Reply:
Both interpretations work. I tend to think of a minus and plus normal, though others only consider that they have a normal